src/mapper/pkg/resolvers/schema.helpers.resolvers.go

```diff
@@ -378,22 +378,25 @@ func (r *Resolver) handleReportTCPCaptureResults(ctx context.Context, results mo
 	for _, captureItem := range results.Results {
```

Comment:
rather than a bunch of `if error != nil { log error and continue }`, consider moving the block inside this `if` to a dedicated function that would return the error. The caller (this for loop) could handle the "log and (implicitly) continue" logic at a single place, making it more natural to add more cases here.